controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -394,56 +393,20 @@ stream {
         server 127.0.0.1:442;
     }
 
-    {{ range $i, $passthrough := .passthroughUpstreams }}
-    upstream {{ $passthrough.Name }}-{{ $i }}-pt {
-        {{ range $server := $passthrough.Endpoints }}server {{ $server.Address }}:{{ $server.Port }};
-        {{ end }}
-    }
-    {{ end }}
+    {{ buildSSPassthroughUpstreams $backends .PassthrougBackends }}
 
     server {
         listen                  443;
-
-        {{ if $cfg.useProxyProtocol }}proxy_protocol on;{{ end }}
-
+        {{ if $cfg.UseProxyProtocol }}proxy_protocol on;{{ end }}
         proxy_pass              $stream_upstream;
         ssl_preread             on;
     }
 
-# TCP services
-{{ range $i, $tcpServer := .tcpUpstreams }}
-    upstream tcp-{{ $tcpServer.Upstream.Name }} {
-        {{ range $server := $tcpServer.Upstream.Endpoints }}server {{ $server.Address }}:{{ $server.Port }};
-        {{ end }}
-    }
-    
-    server {
-        listen {{ $tcpServer.Path }};
-        proxy_connect_timeout  {{ $tcpServer.Proxy.ConnectTimeout }}s;
-        proxy_timeout          {{ $tcpServer.Proxy.ReadTimeout }}s;
-        proxy_pass             tcp-{{ $tcpServer.Upstream.Name }};
-    }
-{{ end }}
-
-# UDP services
-{{ range $i, $udpServer := .udpUpstreams }}
-    upstream udp-{{ $udpServer.Upstream.Name }} {
-        {{ range $server := $udpServer.Upstream.Endpoints }}server {{ $server.Address }}:{{ $server.Port }};
-        {{ end }}
-    }
-    
-    server {
-        listen {{ $udpServer.Path }} udp;
-        proxy_timeout          10s;
-        proxy_responses        1;
-        proxy_pass             udp-{{ $udpServer.Upstream.Name }};
-    }
-{{ end }}
```

Comment:
@aledbf sorry for bumping up old stuff, but I can't figure out why TCP services and UDP services were removed in this commit.  
In current `master` the TCP & UDP services are [present in template config](https://github.com/kubernetes/ingress/blob/f90e9eeb7c64b7679038e99d5719a69fab270c8e/controllers/nginx/pkg/config/config.go#L273-L274) and are [passed to template rendering](https://github.com/kubernetes/ingress/blob/f90e9eeb7c64b7679038e99d5719a69fab270c8e/controllers/nginx/pkg/cmd/controller/nginx.go#L296-L297), but [missed in `nginx.tmpl` at all](https://github.com/kubernetes/ingress/blob/f90e9eeb7c64b7679038e99d5719a69fab270c8e/controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl).